app/reducers/history.ts

```diff
@@ -0,0 +1,40 @@
+import { initialAppState } from './initialAppState';
+import { makeValidAddressBarUrl } from '$Utils/urlHelpers';
+import { TYPES as TABS_TYPES } from '$Actions/tabs_actions';
+import { logger } from '$Logger';
+
+const initialState = initialAppState.history;
+
+const updateHistory = ( state, payload ) => {
+    if ( payload.url ) {
```

Comment:
if no url? should we throw an error on calling that action? To alert whoever is calling the action they've done it wrong?